README.md

```diff
@@ -0,0 +1,279 @@
+<a name="readme-top"></a>
+
+<!--
```

Comment:
![image](https://github.com/Dadadon/hello-rails-react/assets/93039359/bdf56836-e004-411f-ab00-084870446b13)

- [ ] Kindly change your base branch from master to dev since you need to follow the Gitflow 🤔